tiled/server/schemas.py

```diff
@@ -294,6 +294,7 @@ class DeviceCode(pydantic.BaseModel):
 class PrincipalType(str, enum.Enum):
     user = "user"
     service = "service"
+    jwt_token = "jwt_token"
```

Comment:
Can we workshop this name? Principals that are `user` type can also use JWT, so this might be confusing. What if we called it `external` or something like that?